addons/stock/static/src/picking_type_dashboard_graph/picking_type_dashboard_graph_field.scss

```diff
@@ -1,3 +1,9 @@
 .o_field_picking_type_dashboard_graph .o_dashboard_graph {
     margin-bottom: 0;
+    margin-left: -16px;
+    margin-right: -16px;
+
+    canvas {
+        height: 150px;
+    }
```

Comment:
Is it needed ? I saw the canvas already has a height fixed to 150 px directly on the HTML node itself

Reply:
You're right, I just copied the whole style from Accounting I think, to make sure we end up with the same look ;) I will remove this height style, thanks for noticing :+1: